PocketCastsTests/Tests/IAP/IAPHelperTests.swift

```diff
@@ -28,7 +28,7 @@ final class IAPHelperTests: XCTestCase {
         session.clearTransactions()
         SKPaymentQueue.default().remove(helper)
     }
-
+/*
```

Comment:
Does this need to be commented out when the whole file is disabled from the test plan?